cmd/crio/main.go

```diff
@@ -220,6 +223,10 @@ func main() {
 			Name:  "insecure-registry",
 			Usage: "whether to disable TLS verification for the given registry",
 		},
+		cli.StringSliceFlag{
+			Name:  "registry",
+			Usage: "list of registies used when pulling unqualified images",
```

Comment:
Registry to be prepended when pulling unqualified images, can be specified multiple times.